src/marketing/email-template-list.md

```diff
@@ -96,51 +96,50 @@ The list of email templates is organized alphabetically by Magento module.
 {: .template-table }
 |Template|Configuration path|
 |--- |--- |
-| Gift Card(s) Purchase | **Page:** <br/>**Section:** <br/>**Field:** |
+| Gift Card(s) Purchase | **Page:** Sales > [Gift Cards]({% link catalog/product-gift-card.md %})<br/>**Section:** Gift Card Email Settings<br/>**Field:** Gift Card Notification Email Template |
 
 ## Magento_GiftCardAccount
 
 {: .template-table }
 |Template|Configuration path|
 |--- |--- |
-| Gift Card Code/Balance | **Page:** <br/>**Section:** <br/>**Field:** |
+| Gift Card Code/Balance | **Page:** Sales > [Gift Cards]({% link catalog/product-gift-card.md %})<br/>**Section:** Email Sent from Gift Card Account Management<br/>**Field:** Gift Card Template |
 
 ## Magento_GiftRegistry
 
 {: .template-table }
 |Template|Configuration path|
 |--- |--- |
-| New Registry | **Page:** <br/>**Section:** <br/>**Field:** |
-| Registry Sharing | **Page:** <br/>**Section:** <br/>**Field:** |
-| Registry Update | **Page:** <br/>**Section:** <br/>**Field:** |
+| New Registry | **Page:** Customers > [Gift Registry]({% link configuration/customers/gift-registry.md %}) <br/>**Section:** Owner Notification<br/>**Field:** Email Template |
+| Registry Sharing | **Page:** Customers > [Gift Registry]({% link configuration/customers/gift-registry.md %}) <br/>**Section:** Gift Registry Sharing<br/>**Field:** Email Template |
+| Registry Update | **Page:** Customers > [Gift Registry]({% link configuration/customers/gift-registry.md %}) <br/>**Section:** Gift Registry Update<br/>**Field:** Email Template |
 
 ## Magento_InventoryInStorePickupSales
 
 {: .template-table }
 |Template|Configuration path|
 |--- |--- |
-| Order is Ready for Pickup | **Page:** <br/>**Section:** <br/>**Field:** |
-| Order is Ready for Pickup For Guest | **Page:** <br/>**Section:** <br/>**Field:** |
+| Order is Ready for Pickup | **Page:** Sales > [Sales Emails]({% link configuration/sales/sales-emails.md %}) <br/>**Section:** Order Ready For Pickup in Store<br/>**Field:** Order Ready For Pickup Email Template |
+| Order is Ready for Pickup For Guest | **Page:** Sales > [Sales Emails]({% link configuration/sales/sales-emails.md %}) <br/>**Section:** Order Ready For Pickup in Store<br/>**Field:** Order Ready For Pickup Email Template for Guest|
 
 ## Magento_Invitation
 
 {: .template-table }
 |Template|Configuration path|
 |--- |--- |
-| Customer Invitation | **Page:** <br/>**Section:** <br/>**Field:** |
+| Customer Invitation | **Page:** Sales > [Sales Emails]({% link configuration/customers/invitations..md %})<br/>**Section:** Email<br/>**Field:** Customer Invitation Email Template |
```

Comment:
I am not sure on the "Sales Emails" link that is going to the Customers invitations page here. Was this intentional?

Reply:
Hi @leslietilling , thank you for review. Yes you are right :) My mistake. I have changed this to -` | Customer Invitation | **Page:** Customers > [Invitation]({% link configuration/customers/invitations.md %})<br/>**Section:** Email<br/>**Field:** Customer Invitation Email Template | `
Please approve☺️